requirements_optional.txt

```
@@ -1,3 +1,3 @@
-pillow>4.0.0,<10.0.0
+pillow>=10.0.1
```

I actually want to move these to be exact versions, so may as well make that here
```suggestion
pillow==10.0.1
```

---

@jesserockz , I agree pinning dependencies to exact versions is great, I initially followed same semantics as others under `requirements_optional.txt. Will update the PR shortly.